### PR TITLE
test: add tests for studentsSubjects

### PR DESCRIPTION
**Pull request by rajat-ws**
### Ticket Link

---

### Related Links

---

### Description

- added tests for studentsSubjects queries and mutations.

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### CODE REVIEW

File: server/gql/models/tests/studentsSubjects/query.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    expect(result).toEqual(
+      expect.objectContaining({
+        id: studentsSubjectsTable[0].id,
+        studentId: studentsSubjectsTable[0].studentId.toString(),
+        subjectId: studentsSubjectsTable[0].subjectId.toString()
+      })
+    );
+  });
+});
```
Comment (praveenkumar1798) [inline server/gql/models/tests/studentsSubjects/query.test.js:34]: I think we should also test for the shape of the response

File: server/gql/models/tests/studentsSubjects/mutation.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  it('should have a mutation to create a new studentSubject', async () => {
+    const res = await getResponse(createStudentSubjectMutation);
+    const result = get(res, 'body.data.createStudentsSubject');
+    expect(result).toMatchObject({
+      id: '1',
+      studentId: '1',
+      subjectId: '1'
+    });
+  });
+});
```
Comment (praveenkumar1798) [inline server/gql/models/tests/studentsSubjects/mutation.test.js:84]: I think we should also test for update and delete mutation as well. update and delete are exposed right ?
Please do the same everywhere

Comment (rajat-ws) [reply server/gql/models/tests/studentsSubjects/mutation.test.js:84]: Updated accordingly

File: server/gql/models/tests/studentsSubjects/query.test.js (hunk 1 above)
Comment (rajat-ws) [reply server/gql/models/tests/studentsSubjects/query.test.js:34]: image


Already did that here.

**Review by praveenkumar1798 — APPROVED** [review]

**Pull request merged**